examples/js/controls/OrbitControls.js

```diff
@@ -465,15 +465,10 @@ THREE.OrbitControls = function ( object, domElement ) {
 
 		dollyDelta.subVectors( dollyEnd, dollyStart );
 
-		if ( dollyDelta.y > 0 ) {
-
-			dollyIn( getZoomScale() );
-
-		} else if ( dollyDelta.y < 0 ) {
-
-			dollyOut( getZoomScale() );
-
-		}
+		//Arbitrary lower the zoom scale when damping enabled and zoom called from touch and move :
+		//the zoom happens per event, but while scrolling we call 1 event per gesture,
+		//much more happen for mousemove/touchmove. So they need to scale less.
+		dolly( getZoomScale() * ( scope.enableDamping ? 0.4 : 1 ), dollyDelta.y < 0 );
```

Comment:
This worries me... The fact that you added this line means you were not happy without it.

Is this fine-tuned to the particular platform/OS you tested it against? Or does it work universally?

Reply:
OrbitControls trigger one zoom change for each scroll/mousemove/touchmove event. But this is not expected, because with one finger gesture of ~1-3cm, with a same effort from the user, a mousemove/touchmove will trigger a dozen of events, resulting in a big scale change, while a mousewheel only triggers 4-5 events, resulting in a small scale change. At least on windows/linux for all I know, so setting `zoomSpeed` has always been tricky to me. Demos with orbitControls always ask a lot of mousewheels, or if they don't, then the touchmove/mousemove is too sensible.

I conclude that mousewheel should be given more value (or mousemove/touchmove less value), with an arbitrary coefficient like here, it should be the same for all OS because the problem only is about mousewheel generating fewer events. But I'm wrong to only do it with damping here, this is the same in both modes. 

You say that an action on the mousewheel on MacOS triggers much more events ? Then solving that would indeed get worrysome... But I hope it's clarified at least.

Reply:
A mouse wheel event on macOS can generate perhaps 30 events due to built-in inertia. At least that's my experience.

Reply:
In last commits I propose to remove my arbitrary coefficients for mousewheel/touchmove/mousemove because it seems to deserve a specific issue/PR with more in-depth analysis.